bentoml/cli/bento.py

```diff
@@ -74,35 +75,57 @@ def _print_bento_table(bentos, wide=False):
     _echo(table_display)
 
 
-def _print_bentos_info(bentos, output_type):
+def _print_bentos_info(bentos, output_type, print_location=False):
     if output_type == 'table':
         _print_bento_table(bentos)
     elif output_type == 'wide':
         _print_bento_table(bentos, wide=True)
     else:
         for bento in bentos:
+            if print_location:
+                _echo(bento.uri.uri)
             _print_bento_info(bento, output_type)
 
 
+def _unpack_jq_like_string(bento_dict, json_query):
```

Comment:
`jq` actually supports many other syntax(https://stedolan.github.io/jq/tutorial/), I'd prefer not to expose this to the user if it does not fully support all the possible jq queries(e.g. the syntax for accessing array item).  

Getting the location is a very common task and we don't want to force every user to install jq just for that. But for advanced users who can write jq query, I think installing jq wouldn't be a problem and we should probably just let them use command line `jq` with our json output.